ksa_compliance/templates/e_invoice.xml

```diff
@@ -265,8 +265,8 @@
             {% if item.discount_amount %}
             <cac:AllowanceCharge>
                 <cbc:ChargeIndicator>false</cbc:ChargeIndicator>
-                <cbc:Amount currencyID="{{ invoice.currency_code }}">{{ (item.discount_amount * item.qty) | round(2) }}</cbc:Amount>
-                <cbc:BaseAmount currencyID="{{ invoice.currency_code }}">{{ (item.price_list_rate * item.qty) | round(2) }}</cbc:BaseAmount>
+                <cbc:Amount currencyID="{{ invoice.currency_code }}">{{ item.discount_amount| round(2) }}</cbc:Amount>
```

Comment:
We need to look into the XML specification for ZATCA because I'm certain they have some XML elements to support the tax included scenario, which may simplify our implementation.

Reply:
Ok I will take a look and try to figure out.